sponsors.md

```diff
@@ -79,3 +77,17 @@ Thank you to Enrico, who provided ARM and ARM64 could servers for our package bu
 
 <div class="lead text-left"><a href="http://www.heystephenwood.com">Hey Stephen Wood</a></div>
 Thank you to Stephen, who provided a server for our continuous integration system.
+
+### Rackspace
+
+<div class="lead text-left"><a href="https://www.rackspace.com"><img src="/static/images/sponsors/rackspace.jpg" /></a></div>
+Thank you to Rackspace, who provide infrastructure on Rackspace Cloud to power our continuous integration system and automated testing.
+
+### oVirt
+
+<div class="lead text-left"><a href="http://www.ovirt.org"><img src="/static/images/sponsors/oVirt.svg" height="100px"/></a></div>
+Thank you to oVirt, who provide servers for our continuous integration system.
+
+### Scaleway
+<div class="lead text-left"><a href="https://www.scaleway.com/"><img src="/static/images/sponsors/scaleway.svg" height="100px"/></a></div>
+Thank you to Scaleway, who provide ARM servers for building our arm packages.
```

Comment:
```suggestion
Thank you to Scaleway, who provided ARM servers for building our arm packages.
```

Reply:
Technically, we also run Discourse and Redmine there.